virttest/guest_agent.py

```diff
@@ -539,10 +538,10 @@ def set_user_password(self, password, crypted=False, username="root"):
 
         if crypted:
             openssl_cmd = "openssl passwd -crypt %s" % password
-            password = decode_to_text(process.system_output(openssl_cmd)).strip('\n')
+            password = process.system_output(openssl_cmd).strip('\n')
```

Comment:
`password = process.system_output(openssl_cmd).strip('\n')` is bytes
`password.decode()`  is str